group_vars/all.yml

```diff
@@ -54,6 +55,7 @@ scale:
       5039ms:
         rhel7_interfaces:  [enp1s0f0, enp1s0f1, enp2s0f1]
         rhel8_interfaces:  [enp1s0f0, enp1s0f1, enp2s0f1]
+        rhel9_interfaces:  [enp1s0f0, enp1s0f1, enp2s0f1]
```

Comment:
Please add rhel9_interfaces for other hardware types too. I have templates for 5 hardware types(1029p, 1029u tn10rt, 1029u trtp, r640, 6049p) in the following link http://perfscale.perf.lab.eng.bos.redhat.com/pub/schari/osp17_250_node_scale_templates_latest/, which might be useful for reference.

Reply:
I have added the interfaces for the hardware that I have tested OSP17. We can add the other hardware once RHEL 9 is added to foreman on Scalelab and alias lab. @sanjaychari you can create a separate PR for the hardware you have tested on